curator/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM alpine:latest
+
+
+RUN apk --update add python py-pip && \
+    pip install elasticsearch-curator && \
```

Comment:
Formatting a bit off, consider using the following format used in official Dockerfiles:
```dockerfile
RUN foo \
<tab> && bar
```